README.md

```diff
@@ -1,2 +1,13 @@
 # Floats-and-positioning
-Clone of a New York Times article page
+
+This is the third project of the Main HTML/CSS curriculum at Microverse - @microverseinc.
+
+- The goal was to create a clone of [this](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0) New York Times article page.
+- The project was completed using simple html and css.
+
+[Assignment link](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/)
+
+Authors:
+
+- [@BenjaminGarza](https://github.com/BenjaminGarza)
+- [@archemiro](https://github.com/archemiro)
```

Comment:
`AWESOME`

Nice job writing a descriptive `README.md` file for describing what the project does. You can even add other information like assignment link and a screenshot of the webpage. :+1: :star2: